src/htmlContent/main-view.html

```diff
@@ -1,22 +1,22 @@
-<!-- 
```

Comment:
NOTE: you can do `git checkout master src/htmlContent/main-view.html` to fix this.